scrapers/scrape_zh.sh

```diff
@@ -1,14 +1,19 @@
-#!/bin/sh
-set -e
+#!/usr/bin/env python3
 
-DIR="$(cd "$(dirname "$0")" && pwd)"  # " # To make editor happy
+from scrape_common import *
```

Comment:
Please don't `import *` it makes it unclear what's happening and pollutes the namespace (e.g. `filter` is part of the python standard library, so it's not possible to call right now). I'd suggest to use something like `import scrape_common as sc` and then call all the imported functions as `sc.download()` and `sc.filter()`

Reply:
`import *` is fine. It is under our control, so there is nothing wrong with it.

Reply:
It's considered an anti-pattern that's super easy to avoid in this case. Please use `import scrape_common as sc`.

As said before: you are blocking the use of the standard library function `filter`, which is **not** obvious for others developers.

Reply:
It is not an anti-pattern. Otherwise this construct simply would not be in the language. It is only anti-pattern when you do `import *` from a package you don't have control off, and there is a risk of a new function/class overriding somebodies else class and breaking it in the future. There are marginal argument to be made about doing `import *` on many modules, which would make it harder to figure out which module each function is coming from, but again, that is not the case here.

I don't have time to argue about it tho, so I will change it, even if I strongly disagree with it.